classes/Model.php

```diff
@@ -145,16 +145,16 @@ public static function queryOptions($options, $builder)
 	}
 
 	/**
-	 * @return \Nin\Model
+	 * @return ?\Nin\Model
 	 */
 	public static function findByResult($res)
 	{
 		if($res === false) {
-			return false;
+			return null;
 		}
 		$row = $res->fetch_assoc();
 		if(!$row) {
-			return false;
+			return null;
```

Comment:
Returning null instead of false is a **really big** breaking change! I'd prefer if we can still keep returning false here, to make sure existing checks don't break (for example, `=== false` and `!== false`)